### ISSUE TITLE
Problem building the package

### ISSUE THREAD

**Issue by eamanu** [drewnoakes/fix-decoder#17]
Hello, 

I am not a js expert. i am trying to build fix-decoder but when I run ```gulp``` I have the next error:
```
gulp[14802]: ../src/node_contextify.cc:635:static void node::contextify::ContextifyScript::New(const v8::FunctionCallbackInfo<v8::Value>&): Assertion `args[1]->IsString()' failed.
 1: 0x8fa0c0 node::Abort() [gulp]
 2: 0x8fa195  [gulp]
 3: 0x92e5be node::contextify::ContextifyScript::New(v8::FunctionCallbackInfo<v8::Value> const&) [gulp]
 4: 0xb8d2db  [gulp]
 5: 0xb8f272 v8::internal::Builtin_HandleApiCall(int, v8::internal::Object**, v8::internal::Isolate*) [gulp]
 6: 0x379a75b5be1d 
Aborted
```
My tools versions:
```
eamanu@debian:~/dev/fix-decoder$ node --version
v10.16.3
eamanu@debian:~/dev/fix-decoder$ npm --version
6.9.0
eamanu@debian:~/dev/fix-decoder$ yarn --version
1.17.3
eamanu@debian:~/dev/fix-decoder$ gulp --version
CLI version: 2.2.0
Local version: 3.9.1
eamanu@debian:~/dev/fix-decoder$ 
```

**Comment by drewnoakes** [comment]
Sorry I haven't had a chance to look at this repo for a while and don't think I will any time soon. It's quite likely that the tool chain and libraries should be updated. Doing so may fix your problem.

**Comment by whatthefrog** [comment]
Hi @eamanu, sorry I am not too clued up with node/yarn :-(

However, I can advise you to simply `git checkout 237029fd660ee3d0b39a8d77ce992be54a153c39` instead, which commit doesn't have this node/yarn/gulp requirements...

I will create a tag 0.9.0

**Comment by whatthefrog** [comment]
Alternatively, one can download the `1.0.0` built project from https://github.com/drewnoakes/fix-decoder/releases/tag/1.0.0

**Comment by drewnoakes** [comment]
I believe @drekbour fixed this in #28.
